package.json

```diff
@@ -1,8 +1,9 @@
 {
-  "name": "sqon",
+  "name": "sqon-builder",
   "version": "2.0.0",
   "description": "SQON creation and manipulation library.",
   "main": "dist/index.js",
+  "types": "dist/index.d.ts",
```

Comment:
this may be superfluous, but guides devs who want to manually explore the types.